exercises/ex3/description.html

```diff
@@ -1,4 +1,4 @@
-<h2>1-bit adder</h2>
+<h2>1-bit adder (half-adder)</h2>
```

Comment:
Not that big of a deal, but in a PR you should avoid tackling other things than the main goal of your PR. I'll teach you Git Branches ASAP 😉